NEWS.rst

```diff
@@ -35,6 +35,7 @@ possible, especially the following contributors:
 - Michiel de Hoon
 - Peter Cock
 - Sergio Valqui
+- Jakub Lipinski (first contribution)
```

Comment:
Can you move your name up (alphabetic by first name) please?

Reply:
Done